src/lib/util.js

```diff
@@ -990,10 +990,14 @@ function getInterfaceTreeInfo(iface, interfaces) {
   while (iface) {
     for (const [tree, nodes] of Object.entries(trees)) {
       if (nodes.includes(iface)) {
+        // Depth is last occurrence of iface in the array
+        // (but "findLastIndex" is not supported in Node.js)
+        const depth = nodes.length - 1 - nodes.slice().reverse().findIndex(i => i === iface);
```

Comment:
```suggestion
        const depth = nodes.length - 1 - nodes.lastIndexOf(iface);
```
we don't need `findLastIndex` when `lastIndexOf` suffices (and is supported in node)